app/layout.tsx

```diff
@@ -1,7 +1,9 @@
 import { Roboto } from 'next/font/google'
 import './globals.css'
-import { ApolloWrapper } from '@/apollo-provider'
-import { Providers } from '@/Providers'
+import { getUuid } from '@/(auth)/uuid'
+import { ApolloWrapper } from '@/providers/apollo-provider'
+import { Providers } from '@/providers/chakra-providers'
```

Comment:
Could you also change the name of function to `ChakraProvider` ?　And file name should be singlar!